app/presenters/data/mock-bill-run.presenter.js

```diff
@@ -0,0 +1,205 @@
+      netAmount,
+      billingInvoiceId: id,
+      invoiceAccountNumber: account,
+      invoiceNumber: number
+    } = billingInvoice
+
+    return {
+      id,
+      account,
+      number,
+      accountAddress,
+      contact,
+      isWaterCompany: billingInvoiceLicences[0].licence.isWaterUndertaker,
+      credit: formatNumberAsMoney(convertPenceToPounds(creditNoteValue)),
+      debit: formatNumberAsMoney(convertPenceToPounds(invoiceValue)),
+      netTotal: formatNumberAsMoney(convertPenceToPounds(netAmount)),
+      licences: _formatBillingInvoiceLicences(billingInvoiceLicences)
+    }
+  })
+}
+
+function _formatBillingInvoiceLicences (billingInvoiceLicences) {
+  return billingInvoiceLicences.map((billingInvoiceLicence) => {
+    const {
+      billingTransactions,
+      credit,
+      debit,
+      netTotal,
+      licenceHolder,
+      billingInvoiceLicenceId: id,
+      licenceRef: licence
+    } = billingInvoiceLicence
+
+    return {
+      id,
+      licence,
+      licenceStartDate: billingInvoiceLicence.licence.startDate,
+      licenceHolder,
+      credit: formatNumberAsMoney(convertPenceToPounds(credit)),
+      debit: formatNumberAsMoney(convertPenceToPounds(debit)),
+      netTotal: formatNumberAsMoney(convertPenceToPounds(netTotal)),
+      transactions: _formatBillingTransactions(billingTransactions)
+    }
+  })
+}
+
+function _formatBillingTransactions (billingTransactions) {
+  return billingTransactions.map((billingTransaction) => {
+    const {
+      authorisedDays,
+      billableDays,
+      chargeCategoryCode,
+      chargeElement,
+      chargeType,
+      endDate,
+      grossValuesCalculated,
+      isCredit,
+      netAmount,
+      startDate,
+      billableQuantity: chargeQuantity,
+      chargeCategoryDescription: chargeDescription,
+      description: lineDescription
+    } = billingTransaction
+
+    return {
+      type: chargeType === 'standard' ? 'Water abstraction charge' : 'Compensation charge',
+      lineDescription,
+      billableDays,
+      authorisedDays,
+      chargeQuantity,
+      credit: isCredit ? formatNumberAsMoney(convertPenceToPounds(netAmount)) : '0.00',
+      debit: isCredit ? '0.00' : formatNumberAsMoney(convertPenceToPounds(netAmount)),
+      chargePeriod: `${formatLongDate(startDate)} to ${formatLongDate(endDate)}`,
+      chargeRefNumber: `${chargeCategoryCode} (${formatNumberAsMoney(grossValuesCalculated.baselineCharge, true)})`,
+      chargeDescription,
+      addCharges: _formatAdditionalCharges(billingTransaction),
+      adjustments: _formatAdjustments(chargeElement),
+      elements: _formatChargePurposes(chargeElement.chargePurposes)
+    }
+  })
+}
+
+function _formatChargePurposes (chargePurposes) {
+  return chargePurposes.map((chargePurpose) => {
+    const {
+      purposesUse,
+      abstractionPeriodStartDay: startDay,
+      abstractionPeriodStartMonth: startMonth,
+      abstractionPeriodEndDay: endDay,
+      abstractionPeriodEndMonth: endMonth,
```

Comment:
Do we need to alias these seeing as we're passing them to a function and not using them directly? Or do we want to keep that cos if we don't alias them, our call to the function would be too big for one line?